db/migrate/20171213183910_create_user_requests.rb

```diff
@@ -0,0 +1,14 @@
+class CreateUserRequests < ActiveRecord::Migration
+  def change
+    create_table :user_requests do |t|
+      t.string :type, null: false
+      t.references :user, index: true, foreign_key: true, null: false
```

Comment:
in generally, i don't use foreign_keys, but they can be useful. curious why you chose to add it for the user but not for the reviewer column ?

Reply:
this was a function of using `references`when i did created this migration from the command line. (it automatically uses a foreign key and i didn't think about it too closely.) i realized later i probably didn't want/need this since we don't refer to `user.user_requests` and i wouldn't really need the foreign key when/if we did. once i realized this i'd already run the migration and was a bit too lazy to change it. i *did* think about this when adding the reviewer field and didn't add a `reference` for that association.

Reply:
gotcha.